src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -269,6 +268,12 @@ private static void PromptOptionalProperties<T>(T manifest)
 
             foreach (var property in optionalProperties)
             {
+                if (property.Name == nameof(DefaultLocaleManifest.Agreements) &&
+                    !Prompt.Confirm(Resources.EditAgreements_Message))

```

Comment:
Is field a class method?

Reply:
Checks if field is a list of class, rather than using property name.